server/knexfile.js

```diff
@@ -0,0 +1,44 @@
+// Update with your config settings.
+
+module.exports = {
```

Comment:
Should these be updated with the database config settings in `server/config/index.js`?